keyboards/flx/virgo/keymaps/default/keymap.c

```diff
@@ -16,21 +16,37 @@
 #include QMK_KEYBOARD_H
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-    [0] = LAYOUT(
-        KC_ESC,      KC_F1,   KC_F2,   KC_F3,   KC_F4,        KC_F5,                     KC_F6,   KC_F7,   KC_F8,       KC_F9,   KC_F10,  KC_F11,  KC_F12,         KC_PSCR, KC_SLCK, KC_PAUS,
+    [0] = LAYOUT_all(
```

Comment:
In the event that you only have 1 layout, such as one here, it makes more sense to call it LAYOUT rather than LAYOUT_all.

Reply:
Noted: I was taking lead off other keyboards submissions, shall I change this?